packages/gatsby-plugin-clerk/src/constants.ts

```diff
@@ -1,6 +1,8 @@
+import { apiUrlFromPublishableKey } from '@clerk/shared';
```

Comment:
❓ shouldn't we use the `import apiUrlFromPublishableKey from '@clerk/shared/apiUrlFromPublishableKey'` instead?

Reply:
why?